PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -1387,7 +1425,7 @@ def ak4JetReclustering(self,process, pfCandCollection, patMetModuleSequence, pos
                         bottomCollection = cms.InputTag("tmpPFCandCollPtr"+postfix) )
                         )
                 pfCandColl = cms.InputTag("pfNoPileUpJME"+postfix)
-                   
```

Comment:
The part right after "pfCHS = cms.EDFilter" was implemented differently in the earlier version of this PR
(the pfCHS object was manipulated and than was used to override pfNoPileUpJME+postfix after the ``` if self._parameters["onMiniAOD"] ``` conditional)

Looking at 9X implementation, which is most similar in context lines , it seems like the following is missing for the onMiniAOD case
```
                setattr(process,"pfNoPileUpJME"+postfix,pfCHS)
                pfCandColl = cms.InputTag("pfNoPileUpJME"+postfix)
```